src/NUnitFramework/framework/Constraints/ContainsConstraint.cs

```diff
@@ -18,6 +18,7 @@ public class ContainsConstraint : Constraint
         private Constraint? _realConstraint;
         private bool _ignoreCase;
         private bool _ignoreWhiteSpace;
+        private bool _NormalizeLineEndings;
```

Comment:
I guess I'll have to fix this copypasta. I can do this after the first round of reviews.

```suggestion
        private bool _normalizeLineEndings;
```